### DISCUSSION TITLE
2023-07-25

### DISCUSSION THREAD

**Discussion by aak2075** [woowacourse-teams/2023-haru-study#133] [daily_meeting]
### 오늘의 할 일

- 서비스 론칭 완료 및 피드백 반영사항 기능 구현 (~ 1주차 화요일)
    - 프론트, 백 미구현 기능 구현 및 리팩토링 진행
    - 프론트, 백 서비스 연동 완료
    - ~~cloudFlare HTTPS 적용해서 서비스 배포~~
    - 배포 DB 드랍방지 설정
    - ~~API 문서화 진행하기~~
    - 정식 론칭 서비스 배포 완료 (HTTPS 적용)

### 내일의 할 일

- 질문 한 두개만 빼고 선택으로 바꾸기
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 시간 선택 범위 확대하기
- 소켓, 폴링 양방향 통신 어떻게 할지

### 언젠가 해야할 일

- 기획 변경된 이유 정리
- DNS 어떻게 할지

**Comment by jaehee329** [comment]
### 체크인
6점 - 조금 적게 잤어요

### 체크아웃
5점 - 이게 체크아웃? 언제 갈까요 오늘은

**Reply by aak2075** [reply]
뒷공부 하셨나요

**Reply by jaehee329** [reply]
그럴리가요

**Comment by aak2075** [comment]
### 체크인

10점 - 버스타고 출근하니까 너무 편합니다

### 체크아웃

10점 - 블로그도 썼고 https도 적용하고 한게 많아서 좋네요

**Comment by woosung1223** [comment]
### 체크인
7점 - 자다가 더워서 두세번 정도 깨서 -3점입니다..

### 체크아웃
6점 - 오랜만에 고민을 조금 많이 했더니 머리가 아주 뜨끈합니다

**Comment by woo-jk** [comment]
### 체크인
9점 - 나쁘지 않은 하루의 시작~! 하지만 할게 너무 많아서 -1

### 체크아웃
8점 - 예상한 기능 구현이 너무 오래걸린다🥲🥲🥲🥲🥲🥲🥲🥲🥲🥲🥲🥲🥲🥲🥲

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 아침에 일찍 일어나서 많은 걸 한 것 같아서 기분은 좋은데 살짝 졸려서 -1점입니당.

### 체크아웃
7점 - 더 피곤해지고 이력서 생각하니 조금 심란해졌네요 ㅋㅋㅋ...

**Comment by yeopto** [comment]
### 체크인

5점 - 새벽에 어깨가 잠 못 잘정도로 아파서 컨디션이 별로 안좋네요..

### 체크아웃

5점 - 컨디션 난조가 그대로입니다..

**Reply by woosung1223** [reply]
얼마나 총을 열심히 쏘신겁니까..
